lib/ansible/module_utils/common/text/converters.py

```diff
@@ -7,6 +7,7 @@
 
 import codecs
 import json
+import typing as t
```

Comment:
Didn't we settle on `_t`?
```suggestion
import typing as _t
```

Reply:
Not that I am aware of, but I do see it used some:

```
     303 import typing as t
      21 import typing as _t
```

Reply:
we did, but we really need to either start including these decisions in sanity checks or in a style guide, otherwise we will keep having these conversations

Reply:
Yeah, it seems like we're gravitating towards `_t` but not sure if it's properly recorded anywhere.

I was looking at https://github.com/sirosen/flake8-typing-as-t for linting but since we don't use flake8, we'd need to reimplement it for pylint.